packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -207,4 +239,12 @@ export class FormSchemaService {
 
     return [...results];
   }
+
+  private getCachedTranslationsByUuid(translationUuid: string): any {
+    return this.localStorage.getObject(translationUuid);
```

Comment:
Could we cache this in a store rather than localStorage? LocalStorage has some nice properties, but it's also going to create a caching headache here (what happens when the translations are updated on the backend but the user's local storage has a cached copy?)